services/QuillLMS/client/app/bundles/Shared/components/shared/tooltip.tsx

```diff
@@ -71,6 +73,7 @@ export class Tooltip extends React.Component<TooltipProps, {}> {
       >
         <span
           className={`${tooltipTriggerTextClass}`}
+          onClick={onMobile() ? this.handleTooltipClick : () => false} // eslint-disable-next-line react/jsx-no-bind
```

Comment:
If you just do the `onMobile` check inside of `handleTooltipClick` you don't need the bind here.